kind/actions/kind.go

```diff
@@ -24,6 +24,7 @@ import (
 	"os"
 	"path/filepath"
 	"regexp"
+	constkind "sigs.k8s.io/cluster-api-provider-docker/kind/constants"
```

Comment:
The imports are broken up into sections and alphabetized:

```
* first the standard library

* then all the non stdlib imports
```

please run `goimports` over this file